errors.h

```diff
@@ -66,7 +66,7 @@ namespace DS
     if (!(cond)) DS::AssertionFailure(#cond, __FILE__, __LINE__)
 
 #ifdef DEBUG
-#define DEBUG_printf(...)   printf(__VA_ARGS__)
+#define DEBUG_printf(...)   ST::printf(__VA_ARGS__)
```

Comment:
Hope nobody used a single `{` for non formatting purposes. (That has bitten me plenty of times in my ST-consuming code in this same situation.)